src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -314,4 +315,12 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
         log.debug("Notification scheduled successfully");
     }
 
+    @Override
+    public void handleKedaInvocationForMediaRequests() {
+        Optional<MediaRequestEntity> openAudioRequests = mediaRequestService.getOldestMediaRequestByStatus(AudioRequestStatus.OPEN);
+        if (openAudioRequests.isPresent()) {
+            processAudioRequest(openAudioRequests.get().getId());
```

Comment:
Since we get an Optional from `getOldestMediaRequestByStatus()`, a functional approach is a little cleaner here, e.g:

```java
        mediaRequestService.getOldestMediaRequestByStatus(AudioRequestStatus.OPEN)
            .ifPresent(openMediaRequests -> processAudioRequest(openMediaRequests.getId()));
```